source/about/mattermost-v10-changelog.md

```diff
@@ -7,6 +7,79 @@ Support for Mattermost Server v9.5 [Extended Support Release](https://docs.matte
+     - ``CacheSettings.RedisDB``: The database of the Redis host. Typically ``0``.
+     - ``CacheSettings.DisableClientCache``: This can be set to ``true`` if you decide to disable the client-side cache of Redis. Typically there is no need to do this in production, and this is mainly used as a test option.
+ - Plugins are now allowed to add Support Packet data without user interface elements.
+ - Improved the detection of the mobile app operating system as stored in the **Sessions** table.
+
+### Bug Fixes
+ - Fixed an issue where imported replies were missing their reactions.
+ - Fixed an issue with how links in Markdown headings are displayed in the Threads list.
+ - Fixed an issue where marking a channel as read wouldn't persist through a refresh.
+ - Fixed a warning in the Support Packet about an unreadable LDAP server even if LDAP was disabled.
+ - Fixed an issue where multiple timezones were highlighted when selecting certain timezones.
+ - Fixed an issue where unread messages on other teams would not appear after the application reconnected to the server.
+ - Fixed an issue where the scrollbar was not clickable when there was a toaster.
+ - Fixed an issue when pressing **Page Up** or **Page Down** on a long message (scrollable) with the right sidebar open.
+ - Fixed an issue with incorrect reporting in the **Server Updates** section in **System Console > Workspace Optimizations**.
+ - Fixed an issue where EXIF rotated image previews did not have the correct size.
+ - Fixed an issue where the search input field in the emoji picker did not accept uppercase letters.
+ - Fixed an issue where imported replies were missing their reactions.
+ - Fixed an issue where System Admins could not pull posts from Direct Message channels that they were not in.
+
+### config.json
+New setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
+
+#### Changes to all plans:
+- Under ``LocalizationSettings`` in ``config.json``:
+  - Added a new ``EnableExperimentalLocales`` configuration setting that controls whether to allow the selection of experimental (e.g., in progress) languages.
+
+#### Changes to Enterprise plans:
+ - Under ``FileSettings`` in ``config.json``:
+   - Added new ``AmazonS3StorageClass`` and ``ExportAmazonS3StorageClass``, both default to ``""`` to preserve the current behavior. Administrators may configure this storage class to the storage class required by their S3 solution.
+
+### API Changes
+ - Added a new query string to exclude threads that are not part of the team ``GET api/v4/users/{user_id:[A-Za-z0-9]+}/teams/{team_id:[A-Za-z0-9]+}/threads``.
+
+### Websocket Event Changes
+ - Added a new ``server_hostname`` field to the websocket ``HELLO`` event.
+
+### Go Version
+ - v10.4 is built with Go ``v1.22.6``.
+
+### Known Issues
```

Comment:
I'm wondering how we triage the known issues -- for example, I feel like the Playbooks issue has been present forever, I don't think anyone is fixing it, and there are probably worse Playbooks issues we could actually list here.